covid19.occ-pla.net/etlMapping.yaml

```diff
@@ -0,0 +1,74 @@
+mappings:
+  - name: covid19
```

Comment:
in our QA we rename the indices with prefix `qa-`
i don't know why we do that but just wanted to mention it in case we should do it here too

Reply:
Thanks! I though about it. And this will be a separate new elastic search cluster. It should be fine